### ISSUE TITLE
Is this dasbboard responsive?

### ISSUE THREAD

**Issue by dubeychandan996** [safak/next-dashboard-ui#27]

**Comment by Jake9953** [comment]
What do you mean by being responsive?

**Comment by gantzoulis** [comment]
Yes it is responsive

**Comment by Jake9953** [comment]
Thank you

**Comment by hetparikh2209** [comment]
No, it is not giving any response after changing the URL to the dashboard then also it is not changing the dashboard features

**Comment by gantzoulis** [comment]
Well responsive does not mean what you say :P Responsive site is when the site in question changes size and/or how it is presented according to the viewer. Eg. changes layout in order to fit any device etc. Well that it is :)

**Comment by dubeychandan996** [comment]
Yes, when i use the template its not responsive on phone or tablet.

**Comment by DrYoussefTemraz** [comment]
i did not use it on the phone or tablet frankly, but i am coding along and find it responsive on my pc screen.

**Comment by basavarajakj** [comment]
Yes it is pretty much responsive on all screens

**Comment by rishinduyohan** [comment]
visit my repo I updated the UI and role based login : https://github.com/rishinduyohan/next-dashboard-ui

**Comment by crabmites-dev** [comment]
Yes it's responsive
